src/test/jtx/Account.h

```diff
@@ -145,6 +146,12 @@ class Account
     std::string human_;  // base58 public key string
 };
 
+static_assert(std::is_default_constructible_v<Account>);
+static_assert(std::is_copy_constructible_v<Account>);
+static_assert(std::is_move_constructible_v<Account>);
+static_assert(std::is_copy_assignable_v<Account>);
+static_assert(std::is_move_assignable_v<Account>);
```

Comment:
I don't see a lot of value is adding static asserts like this. For example, in the rare cases when you make a class that should be default constructable not-default constructable you'll find that out when you try to use it. I don't object too strongly if you really want to add these although I do prefer the code without it.

Reply:
yeah, that's a reasonable critique. i forgot the context with which I began making this change 😅